compiler/plugins/target/CUDA/CMakeLists.txt

```diff
@@ -21,7 +21,7 @@ iree_cc_library(
   NAME
     CUDA
   HDRS
-    "LLVMPasses.h"
+    "SetBlockIdsRangePass.h"
```

Comment:
History shows this file was added and removed a few times, each time declaring different passes 🤔: https://github.com/openxla/iree/commits/f19d0728fa9faea31d585d156adc8140b0063bc2/compiler/src/iree/compiler/Dialect/HAL/Target/CUDA/LLVMPasses.h?browsing_rename_history=true&new_path=compiler/plugins/target/CUDA/LLVMPasses.h&original_branch=main